components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -344,7 +346,7 @@ public static void usage() {
             + "  $ %s -s localhost -u username -w password -d 50 --debug ALL foo.tiff\n"
             + "\n"
             + "For additional information, see:\n"
-            + "http://www.openmicroscopy.org/site/support/omero5/users/command-line-import.html\n"
+            + "http://www.openmicroscopy.org/site/support/omero5.1/users/cli/import.html\n"
```

Comment:
Agreed, I debated that particular point as well for the same reasons. A couple of thoughts:
- on the reverse side, our docs are currently coupled to the patch version (tags) or the minor versions (latest links), rarely to the major version
- due to the CLI refactoring, this URL will be broken when `site/support/omero5` will be redirected to `site/support/omero5.1`. For the 5.0.x line we may have to implement a redirect independently of the  outcome of this PR.
- the CLI import semantics has been modified between 5.0.x and 5.1.x
- more generally, we are not 100% sure `develop` will be either become 5.2 or 6.0 post 5.1.0. The latter option would also imply a review of all the URLs anyways.

I do not have any problem switching back to `site/support/omero5` but it may be a good time to agree on a policy /cc @hflynn